tutorials/networking/webrtc.rst

```diff
@@ -0,0 +1,207 @@
+.. _doc_webrtc:
+
+WebRTC
+======
+
+HTML5, WebSockets, WebRTC
+-------------------------
+
+One of the great feature of Godot is its ability to the export to HTML5/WebAssembly platform, allowing your game to run directly in the browser when a user visit your webpage.
+
+This is a great oppourtunity for both demos and full games, but used to come with some limitations. In the area of networking, browsers used to support only HTTPRequests until few years ago, first WebSocket, then WebRTC, where proposed as a standard.
```

Comment:
oppo**r**tunity
**a** few years ago **when** first WebSocket **and** then WebRTC **were** proposed as **standards**

Reply:
Nevermind, I'll do the copyedit directly on GitHub :)